### ISSUE TITLE
Display entity states

### ISSUE THREAD

**Issue by markmonroy** [andrew-codechimp/ha-menu#40]
Love seeing this app develop so quickly!

My feature suggestion is to have a section of the menu devoted to displaying the state of entities that aren't boolean switches. I'm picturing a sub-menu:

![sample_status](https://user-images.githubusercontent.com/32033503/72754274-575b7980-3b95-11ea-9f71-e3956d65063c.jpg)

Maybe with the option of displaying all state attributes as well.

**Comment by andrew-codechimp** [comment]
Thanks, I’m enjoying working on this. 
Interesting idea but why a submenu rather than entity name - value which would save having to navigate into the entity?

**Comment by markmonroy** [comment]
Good point; that’s probably better for most use cases. I do have a few template sensors that pull attributes from other entities—Now Playing on Apple TV for instance. If I had that in the main menu it would have to be truncated or wrapped in order to avoid cluttering the UI with a long video title.

That’s not something I desperately need, just an example. Most of my entities have states that are 20 characters or less.

**Comment by markmonroy** [comment]
﻿
Good point; that’s probably better for most use cases. I do have a few template sensors that pull attributes from other entities—Now Playing on Apple TV for instance. If I had that in the main menu it would have to be truncated or wrapped in order to avoid cluttering the UI with a long video title.

That’s not something I desperately need, just an example. Most of my entities have states that are 20 characters or less.

**Comment by andrew-codechimp** [comment]
I’ll have a play with displaying state values and see how it looks. 
I’m a little worried about formatting things like temperatures which don’t have their units and making long states useful but it might be workable.

**Comment by mgguinne** [comment]
Hi, Any update on this one plase?  I actually got this sort of working using the following repo. Though I can't get it to display in the meny bar just yet. (via xbar)

https://github.com/petrepa/BitBar-HASS-fetch

**Comment by andrew-codechimp** [comment]
I've not had chance to look at this yet, work has got busy.  It's still definitely on my todo list though.

**Comment by andrew-codechimp** [comment]
I got some time to add this.
Version 2.5.0 now supports sensor.xxx entities within groups, see the readme for an example.

**Comment by mgguinne** [comment]
Nice once thanks for doing that so quickly :)
